test/benchmark/run_benchmark.py

```diff
@@ -0,0 +1,36 @@
+import os
+import random
+import sys
+import logging
+import re
+import uuid
+
+import boto3
+import pytest
+
+from invoke import run
+from invoke.context import Context
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+LOGGER.addHandler(logging.StreamHandler(sys.stdout))
+
+
+def main():
+    # Run this script from the root directory 'serve', it changes directory below as required
+    os.chdir(os.path.join(os.getcwd(), "test", "benchmark"))
+
+    execution_id = f"ts-benchmark-run-{str(uuid.uuid4())}"
```

Comment:
Suggestion. : Date prefix if it would help.

Reply:
The uuid4() is based on the current timestamp as well: https://docs.python.org/3/library/uuid.html#uuid.UUID.fields. This can be made more human readable, making a note. https://github.com/pytorch/serve/issues/1067